### PR TITLE
Am 2721 case role validation drools staff judiciary

### PR DESCRIPTION
**Pull request by abul-syed**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/AM-2721

### Change description ###

ST_CIC_WA - Add CASE role validation drool rules in RAS application for STAFF And JUDICIARY

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/DroolBase.java [hunk 1]
```diff
@@ -16,10 +16,7 @@
 
 import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.mock;
-import static uk.gov.hmcts.reform.roleassignment.feignclients.configuration.DataStoreApiFallback.PRIVATE_LAW_CASE_ID;
-import static uk.gov.hmcts.reform.roleassignment.feignclients.configuration.DataStoreApiFallback.SSCS_CASE_ID;
-import static uk.gov.hmcts.reform.roleassignment.feignclients.configuration.DataStoreApiFallback.CIVIL_CASE_ID;
-import static uk.gov.hmcts.reform.roleassignment.feignclients.configuration.DataStoreApiFallback.PUBLIC_LAW_CASE_ID;
+import static uk.gov.hmcts.reform.roleassignment.feignclients.configuration.DataStoreApiFallback.*;
```
Comment (mikebrownccd) [inline src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/DroolBase.java:19]: * import not liked by checkstyle

Comment (abul-syed) [reply src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/DroolBase.java]: imports updated

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Review by mattnayler — COMMENTED** [review]
Reviewed alongside #1916 and current documented mappings in config.

**Pull request merged**